core/store/orm/orm.go

```diff
@@ -1173,6 +1175,44 @@ func (orm *ORM) FindLogCursor(name string) (models.LogCursor, error) {
 	return lc, err
 }
 
+// HasConsumedLog reports whether the given consumer had already consumed the given log
+func (orm *ORM) HasConsumedLog(rawLog eth.RawLog, consumer models.LogConsumer) (bool, error) {
```

Comment:
Do we really want to pull `eth` into `orm.go`?

Seems like this could sit somewhre higher up.

Reply:
For other reasons, we're already planning to move `eth.Log` into the `models` package.  Will this address your concern?